snippets/base/forms.py

```diff
@@ -119,6 +121,14 @@ class BaseSnippetAdminForm(forms.ModelForm):
 class SnippetAdminForm(BaseSnippetAdminForm):
     template = forms.ModelChoiceField(queryset=SnippetTemplate.objects.exclude(hidden=True),
                                       widget=TemplateSelect)
+    client_option_version_lower_bound = forms.ChoiceField(
+        label='Firefox Version at least',
+        choices=[('any', 'No limit'),
+                 ('current_release', 'Current release')])
+    client_option_version_upper_bound = forms.ChoiceField(
+        label='Firefox Version at most',
+        choices=[('any', 'No limit'),
+                 ('older_than_current_release', 'Older than current release')])
```

Comment:
Well I am a bit confused here, if `current release == Older than current release`, I would suggest to change the name. If not please ignore :)